--- a/vendor/zed/crates/gpui_linux/Cargo.toml
+++ b/vendor/zed/crates/gpui_linux/Cargo.toml
@@ -56,6 +56,7 @@
 anyhow.workspace = true
 bytemuck = "1"
 collections.workspace = true
+font-kit = { git = "https://github.com/zed-industries/font-kit", rev = "94b0f28166665e8fd2f53ff6d268a14955c82269", package = "zed-font-kit", version = "0.14.1-zed" }
 futures.workspace = true
 gpui.workspace = true
 gpui_util.workspace = true
@@ -64,6 +65,7 @@
 libc.workspace = true
 log.workspace = true
 parking_lot.workspace = true
+pathfinder_geometry.workspace = true
 smallvec.workspace = true
 smol.workspace = true
 strum.workspace = true
